electron31/loong.patch

```diff
@@ -0,0 +1,110 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index f8f04f4..c1b1352 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -456,6 +456,21 @@ prepare() {
+   sed -i "s|@ELECTRON@|${pkgname}|" electron.desktop
+   sed -i "s|@ELECTRON_NAME@|Electron ${_major_ver}|" electron.desktop
+ 
++  # Patching makepkg-source-roller-orig.py
++  cp "${srcdir}/makepkg-source-roller.py" "makepkg-source-roller-new.py"
++  patch "makepkg-source-roller-new.py" -i "${srcdir}/makepkg-source-roller.py.diff"
++  mv -f "makepkg-source-roller-new.py" "makepkg-source-roller.py"
++
++  # Apply in advance to so that makepkg-source-roller.py will not throw error
++  patch -Np1 -d chromium-mirror_third_party_depot_tools -i "${srcdir}/depot_tools-loong64-support.patch"
++  # Get the binary from the npm package
++  npm install @esbuild/linux-loong64@0.14.54
++
++  # Add ` -mcmodel=medium` to CFLAGS etc.
```

Comment:
There is no need to add this in new patches.

See: https://github.com/lcpu-club/devtools-loong64/commit/51f052c6b724140bd69dfc5f7731ec61ea5b9016